keel/gradle.properties

```diff
@@ -7,7 +7,7 @@ liquibaseTaskPrefix=liquibase
 org.gradle.parallel=true
 testContainersVersion=1.15.3
 okHttpVersion=4.5.0
-resilience4jVersion=1.5.0
+resilience4jVersion=2.0.1
```

Comment:
Better to remove this an let spring boot / spinnaker-dependencies decide the version?